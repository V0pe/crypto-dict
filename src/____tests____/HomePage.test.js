import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import GlobeInfo from '../Components/Pages/Homepage';
import store from '../Redux/Configurestore';

it('renders correctly', () => {
  const tree = renderer
    .create(<Router><Provider store={store}><GlobeInfo /></Provider></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
