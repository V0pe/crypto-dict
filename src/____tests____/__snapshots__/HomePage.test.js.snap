// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section>
  <header
    className="header"
  >
    <div>
      <button
        className="logoButton"
        onClick={[Function]}
        type="button"
      >
        <h1>
          <img
            alt="logo"
            className="app-logo"
            src="cryptocurrencies.png"
          />
          CRYPTO
        </h1>
      </button>
    </div>
    <h1>
      Major Crypto Markets
    </h1>
  </header>
  <div
    className="price-filter"
  >
     Filter the Price of Coin: 
    <input
      id="price-range"
      max="9999999999"
      min="0"
      name="price-range"
      onChange={[Function]}
      placeholder="0"
      type="number"
      value={0}
    />
     USD($)
  </div>
  <section>
    <ul
      className="coin-container"
    />
  </section>
</section>
`;
